### DISCUSSION TITLE
Change Discord RPC to listening

### DISCUSSION THREAD

**Discussion by Jackolix** [ciderapp/Cider-2#344] [Feature Requests]
Hey, 
it would be cool if in Discord instead of **Playing** Apple Music, Cider, etc.  displaying **Listening** to Cider, Apple Music

Thanks

**Comment by freehelpdesk** [comment]
Not possible with RPC. It requires a Vencord plugin to change Playing to Listening.

Trust me, We would have already added it if it were possible.

**Comment by Jackolix** [comment]
May i ask why it is not possible? I did once a discord bot where it was easy. But i know that cider is a big project

**Comment by freehelpdesk** [comment]
Its not possible because discord does not allow it.
 Discord doesn't allow the third-party clients to change ActivityType Enum at all, and it is stuck at the Playing Enum for clients

**Comment by freehelpdesk** [comment]
![image](https://github.com/ciderapp/Cider-2/assets/115826724/3c002bb4-b8a5-4cbb-8244-2d4bb362efea)
Here are all of the Enums for the status

**Comment by freehelpdesk** [comment]
![image](https://github.com/ciderapp/Cider-2/assets/115826724/f7df67da-ba76-4a24-95c3-95686e8b6612)

**Comment by Jackolix** [comment]
Oh okay, Thanks for your explaination
